applications/StructuralMechanicsApplication/custom_strategies/custom_schemes/explicit_central_differences_scheme.hpp

```diff
@@ -0,0 +1,922 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
```

Comment:
I recommend to start moving the code to *.cpp. It is a very bad practice to work always in the header, this is one of the main reasons Kratos takes so long to compile, so I recommend to move to a source file

Reply:
I thought this does not work bcs it is templated?

Reply:
you guys are both right. The big question is however if we do need the templates, since i guess the spaces and linear solver is not needed